src/components/pages/career/career-page.tsx

```diff
@@ -11,13 +11,28 @@ import { useTranslation } from 'gatsby-plugin-react-i18next';
 import { SyPersonioJob } from '../../../types';
 import { ImageHero } from '../../content/heroes';
 import { IGatsbyImageData } from 'gatsby-plugin-image';
+import { ImageSpacer } from '../../pages/landingpage/image-spacer';
```

Comment:
Because the `ImageSpacer` is now used not only on the landing page, it makes sense to move the `ImageSpacer` component into the `components/ui` folder

Reply:
Yes, it makes totally sense as you suggested. Thanks I have changed it too.